doc/performance-investigations/0017-upgrade-python-3.9/summary.md

```diff
@@ -0,0 +1,30 @@
+# Title
```

Comment:
The title could be a bit more descriptive 😉

Reply:
🤦